packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,22 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessible label for the checkbox. Defaults to 'label' property value.
+   */
+  ariaLabel?: string;
+
+  /**
+   * ID for element that contains label information for the checkbox.
+   * Defaults to 'label' property value.
```

Comment:
No, what I mean is this property has no default value set in code, yet the comment here implies that it does. Additionally, if it DOES have a default value, its value should be the ID of the label, not the contents of the label.